Lesson1/Dariia-Skoryk/task1_vector/test_vector/tst_testcase.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <gtest/gtest.h>
+#include <gmock/gmock-matchers.h>
+#include "../source/main.h"
+
+using namespace testing;
+
+TEST(testCase, testSet)
+{
+    std::vector<int> firstVector, secondVector, resultVector;
+    EXPECT_TRUE(resultVector == combineVectors(firstVector.begin(), firstVector.end(),
```

Comment:
For debugging purposes, it is always better to split such expressions.
Usually, you won't get any runtime overhead.